lib/utils/system_bars.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+
+/// System bars styling is set
```

Comment:
should be consistent and use `///` for the whole comment

Reply:
Using all /// for whole comment section 👍